source/screens/bookingScreens/BookingSummary.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import styled from "styled-components/native";
+import { BookingItem } from "../../helpers/MockBookingData";
+import icons from "../../helpers/Icons";
+import { Text } from "../../components/atoms";
+import {
+  DateTimeCard,
+  AddressCard,
+  CharacterCard,
+} from "../../components/molecules";
+
+const Container = styled.ScrollView`
+  height: 80%;
+  margin-top: 0px;
+  padding-left: 16px;
+  padding-right: 16px;
+  background-color: ${({ theme }) => theme.colors.neutrals[6]};
+`;
+
+const Title = styled(Text)`
+  margin-top: 50px;
+  margin-bottom: 15px;
+`;
+
+interface BookingSummaryProps {
+  route: any;
+}
+
+const BookingSummary = ({ route }: BookingSummaryProps): JSX.Element => {
+  const routeName = route?.name;
+  const isConfirmation = routeName === "Confirmation";
+
+  const bookingItem = route?.params?.bookingItem as BookingItem;
+  const { administrator } = bookingItem;
+  const timeString =
```

Comment:
you don't need the `+` sign:

```js
const timeString = `${bookingItem.time.startTime.substring(0, 5)} - ${bookingItem.time.endTime.substring(0, 5)}`;
```

Reply:
I added the + because prettier insisted on butchering these two lines into 4 hard to understand lines otherwise. I think we should keep it like this for the sake of readability.